ding_dibs.module

```diff
@@ -386,6 +386,28 @@ function ding_dibs_menu() {
     'file' => 'ding_dibs.reports.inc',
   );
 
+  $items['admin/config/payment/ding_dibs/emails'] = array(
+    'title' => 'Ding DIBS setting emails',
+    'description' => 'Ding DIBS setting emails.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_dibs_setting_emails'),
```

Comment:
This mentions emails a lot but as I understand it is has more to do with reporting? Getting the reports by emails is just one aspect of it.